fabric-common/types/index.d.ts

```diff
@@ -245,6 +245,15 @@ export class Proposal extends ServiceAction {
 	public compareProposalResponseResults(proposalResponses: any[]): boolean;
 }
 
+export class BlockDecoder {
+	constructor();
+	public decode(blockBuf: Buffer): fabproto6.common.Block;
```

Comment:
@jscode017 These methods are static according to the documentation, please update the type definitions accordingly. https://hyperledger.github.io/fabric-sdk-node/release-2.2/BlockDecoder.html

Reply:
OK, fix it.

Reply:
Awesome, cheers!